docs/sources/next/shared/experimental-timers-module.md

```diff
@@ -4,8 +4,8 @@ title: Experimental timers module admonition
 
 {{% admonition type="caution" %}}
 
-Starting on k6 `v0.50`, the experimental module `k6/experimental/timers` has been graduated, and its functionality is now available in the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed in `v0.52.0`.
+The experimental module `k6/experimental/timers` has graduated, and its functionality is now globally available and through the [`k6/timers` module](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-timers/). The `k6/experimental/timers` is deprecated and will be removed.
 
-To migrate your scripts, replace all `k6/experimental/timers` imports with `k6/timers`.
+To migrate your scripts, remove the `k6/experimental/timers` imports or import them using the `k6/timers` module.
```

Comment:
```suggestion
To migrate your scripts, remove the `k6/experimental/timers` imports.
```

I understand importing `k6/timers` would still work, but could we just suggest removing it for simplicity?

Reply:
Yeah - I thin that is better actually. It  is what we will be doing in the cli warning